src/main/java/accesingdatapostgres/api/CarApi.java

```diff
@@ -0,0 +1,47 @@
+package accesingdatapostgres.api;
+
+import accesingdatapostgres.dao.entity.CarEntity;
+import accesingdatapostgres.service.CarService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping(path = "/cars")
+public class CarApi {
+    private final CarService carService;
+
+    @Autowired
+    public CarApi(CarService carService) {
+        this.carService = carService;
+    }
+
+    @PostMapping(path = "/add")
+    public CarEntity addCar(@RequestBody CarEntity carEntity) {
+
+        return this.carService.save(carEntity);
+    }
+
+    @PutMapping(path = "/update")
+    public CarEntity updateCar(@RequestBody CarEntity carEntity) {
```

Comment:
jak zmienisz na CarDto to potrzebne Ci będzie jeszcze id, najlepiej dać w path tak jak zrobiłeś to niżej